### PR TITLE
[[ Documentation ]] Changes to asterisk.lcdoc

### PR DESCRIPTION
**Pull request by asayd**
* Updated summary to mention array multiplication.
* Added html5 as OS.
* Improved examples.
* Added note about operator precedence.
* Removed unneeded reference.

### CODE REVIEW

File: docs/dictionary/operator/asterisk.lcdoc [hunk 1]
```diff
@@ -28,25 +30,29 @@ put tArray * 10 into tProductArray
...
-The operands number2 and number2 are numbers or expressions that
-evaluate to numbers, or arrays containing numbers.
+A number or an expression that evaluates to a number, or an array 
+containing numbers.
 
 Description:
 Use the <*> (times) <operator> to multiply two numbers.
 
+> *Note:* When used in complex arithmetic statements, 
+<operator|operators> like </> follows standard rules of <precedence>.
```
Comment (livecodeali) [inline docs/dictionary/operator/asterisk.lcdoc]: Should be `... like <*> ...` here

Comment (asayd) [reply docs/dictionary/operator/asterisk.lcdoc]: Dangers of copy/paste!

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 38ee48e

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 38ee48e0701e63b570b84954e1ef04f94443184f

**Comment by livecode-vulcan** [comment]
😎 test success 38ee48e0701e63b570b84954e1ef04f94443184f

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/665)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/670)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/665)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/669)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/674)
* try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/666)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/666)

**Pull request merged**